imap_processing/tests/ultra/unit/test_ultra_l1c_pset_bins.py

```diff
@@ -35,6 +36,16 @@ def test_data():
     return v, energy
 
 
+@pytest.fixture(scope="session")
+def test_l1c_data():
+    """Ensure external test data is downloaded before tests run."""
+
+    # Make sure we have the data available here. This test collection gets
+    # skipped at the module level if the mark isn't present. We can't decorate
```

Comment:
This file doesn't have the module level skip so this comment can be removed and the other marks can be utilized still.